component/constraint/model/SurfacePressureModel.inl

```diff
@@ -418,10 +437,12 @@ void SurfacePressureModel<DataTypes>::storeLambda(const ConstraintParams* cParam
 
     d_pressure.setValue(lambda->element(m_columnId));
 
+
     // Compute actual cavity volume and volume growth from updated positions of mechanical
     const VecCoord& position = m_state->read(VecCoordId::position())->getValue();
     d_cavityVolume.setValue(getCavityVolume(position));
     d_volumeGrowth.setValue(d_cavityVolume.getValue()-d_initialCavityVolume.getValue());
+    d_deltaCavityVolume.setValue(d_initialCavityVolume.getValue() - getCavityVolume(position));
```

Comment:
By seeing those two lines side by side I'm wondering, why do we need `d_deltaCavityVolume` as we already have `d_volumeGrowth`?

```
d_volumeGrowth.setValue(d_cavityVolume.getValue()-d_initialCavityVolume.getValue());
d_deltaCavityVolume.setValue(d_initialCavityVolume.getValue() - getCavityVolume(position));
```

Also, I think you don't need this extra computation of volume `getCavityVolume(position)`, it is already done two lines above and the value is stored in `d_cavityVolume`.

Reply:
Ohh, I see! I overlooked it, because in the GUI the volumeGrowth Data is in a different Tab. I will just remove it then.